back/app/Models/Message.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletingScope;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Message extends Model
+{
+    use softDeletes;
+
+    protected $fillable = [
+        'title',
+        'content',
+        'link',
+        'is_active',
+    ];
+
+    protected $casts = [
+        'is_active' => 'boolean',
+    ];
+
+    public static function active()
+    {
+        return static::where('is_active', true)->get();
+    }
```

Comment:
Tu pourras utiliser cette fonction pour récupérer tous les messages qui sont actifs dans la base de données (bien qu'il n'est censé y'en avoir qu'un seul maximum à chaque fois). Tu verras que tu vas devoir supprimer le `->get()` pour pouvoir continuer, je ne t'en dis pas plus pour le moment.
```suggestion
    public static function active()
    {
        return static::where('is_active', true);
    }
```